docs/security.html

```diff
@@ -470,7 +473,10 @@
         By default all principals that don't have an explicit acl that allows access for an operation to a resource are denied. In rare cases where an allow acl is defined that allows access to all but some principal we will have to use the --deny-principal and --deny-host option. For example, if we want to allow all users to Read from Test-topic but only deny User:BadBob from IP 198.51.100.3 we can do so using following commands:
         <pre>bin/kafka-acls.sh --authorizer-properties zookeeper.connect=localhost:2181 --add --allow-principal User:* --allow-host * --deny-principal User:BadBob --deny-host 198.51.100.3 --operation Read --topic Test-topic</pre>
         Note that ``--allow-host`` and ``deny-host`` only support IP addresses (hostnames are not supported).
-        Above examples add acls to a topic by specifying --topic [topic-name] as the resource option. Similarly user can add acls to cluster by specifying --cluster and to a consumer group by specifying --group [group-name].</li>
+        Above examples add acls to a topic by specifying --topic [topic-name] as the resource option. Similarly user can add acls to cluster by specifying --cluster and to a consumer group by specifying --group [group-name].
+        <p></p>
+        When using SSL-based client authentication, the principal value will need to match the client certificate's subject name. For example:
+        <pre>... --allow-principal 'User:C=US,ST=CA,L=Santa Clara,O=org,OU=org,CN=Sriharsha Chintalapani' ...</pre></li>
```

Comment:
A few things:
* It may be better to say "distinguished name (DN)" instead of "subject"
* This is the case for the default principal builder, but users can provide their own principal builders. It may be worth saying that.
* Maybe we should use some made-up name in the certificate instead of a real one?